owncloudData/src/test/java/com/owncloud/android/data/files/repository/OCFileRepositoryTest.kt

```diff
@@ -206,6 +211,36 @@ class OCFileRepositoryTest {
         }
     }
 
+    @Test
+    fun `getFilesLastUsageIsOlderThanGivenTime returns a list of OCFile`() {
+        every {
+            localFileDataSource.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+        } returns emptyList()
+
+        val result = ocFileRepository.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+
+        assertEquals(emptyList<OCFile>(), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+        }
+    }
+
+    @Test
+    fun `getFilesLastUsageIsOlderThanGivenTime returns an empty list when datasource returns an empty list`() {
+        every {
+            localFileDataSource.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+        } returns listOf(OC_FILE)
+
+        val result = ocFileRepository.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+
+        assertEquals(listOf(OC_FILE), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getFilesWithLastUsageOlderThanGivenTime(timeInMilliseconds)
+        }
+    }
```

Comment:
You keep confusing these 2 tests. Watch the name of the test and its content!!

Reply:
Yes... I accidentally mixed them up.